cmd/influx/config.go

```diff
@@ -110,10 +119,9 @@ func (b *cmdConfigBuilder) cmdCreateRunEFn(*cobra.Command, []string) error {
 	}
 
 	p := config.Config{
-		Host:   b.url,
-		Token:  b.token,
-		Org:    b.org,
-		Active: b.active,
+		Host:  b.url,
+		Token: b.token,
+		Org:   b.org,
```

Comment:
why is active dropped when creating a new one?

Reply:
it will be active, when calling the switch later. It is removed, the the previous active logic can be more consistent, you can check the change of create test case for more details

Reply:
the amount of side effects in this PR is scaring me... feels very unobvious what is happening here

Reply:
are you planing on removing the active flag for this command? Otherwise it needs to be added to this config yah?